<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IWExtern</class>
 <widget class="QWidget" name="IWExtern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlotA">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="_DisplayBox_Msg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>761</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>16</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDDisplay_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_y3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_x2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_x1">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_y2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_x3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QwwLed" name="P1_led_y1">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlotB">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDDisplay_1">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwwLed</class>
   <extends>QWidget</extends>
   <header>qwwled.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
